components/admin_console/permission_schemes_settings/guest_permissions_tree/guest_permissions_tree.test.tsx

```diff
@@ -4,27 +4,23 @@
 import React from 'react';
 import {shallow} from 'enzyme';
 
-import GuestPermissionsTree from 'components/admin_console/permission_schemes_settings/guest_permissions_tree/guest_permissions_tree.jsx';
+import {TestHelper} from 'utils/test_helper';
 
+import GuestPermissionsTree from 'components/admin_console/permission_schemes_settings/guest_permissions_tree/guest_permissions_tree';
 import PermissionGroup from 'components/admin_console/permission_schemes_settings/permission_group.jsx';
 
 describe('components/admin_console/permission_schemes_settings/permission_tree', () => {
     const defaultProps = {
         scope: 'channel_scope',
-        config: {
```

Comment:
`config` and related test is removed as there is no `config` prop in the component